Lib/subprocess.py

```diff
@@ -590,7 +592,7 @@ def __init__(self, args, bufsize=-1, executable=None,
                  shell=False, cwd=None, env=None, universal_newlines=False,
                  startupinfo=None, creationflags=0,
                  restore_signals=True, start_new_session=False,
-                 pass_fds=(), *, encoding=None, errors=None):
+                 pass_fds=(), *, encoding=None, errors=None, text=False):
```

Comment:
Change the defaults for `universal_newlines` and `text` from `False` to `None`.

and in the code below, raise an error when both are supplied but disagree:

```python
if (text is not None and universal_newlines is not None
    and bool(universal_newlines) != bool(text)):
    raise SubprocessError('Cannot disambiguate when both text '
                          'and universal_newlines are supplied but different. '
                          'Pass one or the other.')
```